configs/ASRock/X370-Gaming-K4.conf

```diff
@@ -0,0 +1,115 @@
+chip "nct6779-isa-*"
+
+    # Fans
+    label fan1 "Chassis Fan 3 Speed"
+    label fan2 "CPU Fan 1 Speed"
+    label fan3 "CPU Fan 2 Speed"
+    label fan4 "Chassis Fan 1 Speed"
+    label fan5 "Chassis Fan 2 Speed"
+
+    # Temperatures
+    label temp1 "Motherboard"
+    label temp2 "CPU MB"
+    label temp3 "Auxillary"
+    label temp4 "VRM"
+    label temp5 "AUXTIN2"
+    label temp6 "CPU (PECI)" #Almsot always at -25. Not certain what to do about this.
+    
+    # A Thermal BIAS value generated by CPU that includes an offset to drive mainboard Fans to higher speeds.
+    # This value is rounded off to the nearest ~1C and is only reported if "Use Tctrl" has been selected in UEFI as the fan control sensor.
```

Comment:
Please split up all lines longer that 80 characters, it's much easier to read that way.